resources/ThePath.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The whole xml file must be properly formatted (currently the indentations are not according to the agreed formatter rules). Please format the file.